install-kubeadm.bash

```diff
@@ -12,9 +12,31 @@ apt-get update
 
 apt-get install -y docker-ce=17.03.2~ce-0~ubuntu-xenial
 
+COMMON_DOCKER_PROPS='"storage-driver": "overlay2"'
+
+# See if a docker volume device was provided
+if [ "$#" = 1 ]; then
+  mkfs -t ext3 $1
```

Comment:
I'd rather not format this, so that the user can format it before and choose their favorite filesystem

Reply:
we also need to add this to the docs